package.json

```diff
@@ -22,7 +22,11 @@
     "express": "^4.16.4",
     "graphql": "^14.1.1",
     "http": "0.0.0",
-    "lodash": "^4.17.11"
+    "lodash": "^4.17.11",
+    "pg": "^7.10.0",
+    "sequelize": "^5.8.5",
+    "sequelize-cli": "^5.4.0",
+    "sqlite3": "^4.0.6"
```

Comment:
This is not used anymore, right? In that case, please remove it from the package.json.

Reply:
Right, this was removed from the package.json in the more update mn/setdatabase branch.